docs/triggers/shortcut-triggers.md

```diff
@@ -0,0 +1,125 @@
+## Shortcut Triggers
+
+A shortcut trigger is a trigger that activates when a shortcut is clicked in the Slack client. When a shortcut trigger is created, its API response returns a `shortcut_url` which can be pased into the Slack client causing the client to render it as a button. Clicking on this button will activate the associated workflow. A shortcut trigger
+includes the common [trigger attributes](./trigger-basics.md#trigger-types) along with an optional shortcut parameter: 
+
+| Parameter name  | Required?     | Description                                                          |
+| ----------------|:-------------:| ---------------------------------------------------------------------|
+| `shortcut`        | No            | Contains information about the [button text](#shortcut-object) of the shortcut          |
+
+### Shortcut Object
+
+A Shortcut trigger can contain an optional shortcut object which specifies additional details about the shortcut. Currently, the shortcut object is used to specify the button text of the shortcut associated with the trigger and has the following shape:
+
+```ts
+  shortcut?: {
+    button_text: string;
+  };
+```
+
+### Shortcut Data Context Object
+The Shortcut trigger also has access to a data context object which includes information related to the context of the trigger activation. This data object can be used to fill the optional input fields of the trigger being activated. The data context parameters available to a shortcut trigger are as follows:
+```ts
+  'data.user_id': string,
+	'data.channel_id': string, 
+	'data.interactivity': InteractivityValue::T,
+	'data.location': string, //message, file, bookmark
+	'data.message_ts': string,
+	'data.user': UserContextValue::T,
+	'data.action_id': string,
+	'data.block_id': string,
+	'data.bookmark_id': string,
+	'data.canvas_id': string,
+```
+
+The data context can be used in the input parameter as follows:
+
+```ts
+{ 
+ inputs: {
+  a_input_value: {
+    value: "{{data.user_id}}"
+  }
+ }
+}
+```
+
+
+### Shortcut Unfurling 
+A shortcut trigger can be used to create a shortcut which, when clicked or activated, will run through a workflow. When a shortcut trigger is created through the API, it's return object will include a shortcut_url parameter which can be pasted into Slack client and unfurled to reveal a shortcut link. Clicking on this link will run through the associated workflow.
+
+## Usage
+### Example Shortcut Trigger With Shortcut Object
+
+Below are some usage examples of `ShortcutTrigger` objects which can be used in a .ts file to create a `shortcut` trigger through the Hermes CLI, alternatively this object could be passed into a 
```

Comment:
I think we should state this assumption under the `Usage` title and link to the CLI vs Runtime portion of `trigger-basics`